pyproject.toml

```diff
@@ -72,20 +72,20 @@ jsonschema = ">=4.0.0"
 deprecation = "^2.1.0"
```

Comment:
Due to [Github limitation](https://github.com/orgs/community/discussions/4452) I can't comment on the exact line I need to, but on `L69` can be entirely removed now, as it's only needed by Python 3.7

Arguably, we could keep it for people who forcibly install on 3.7, but I think I err on the side of not doing personally, especially given that you've changed other python version dependant installs.